integration/apps/hpl_netlib/build.sh

```diff
@@ -36,6 +36,7 @@ set -e
 # Get helper functions
 source ../build_func.sh
 
+
```

Comment:
???

Reply:
:) I put in MKL version and removed it, apparently whitespace remained after that.